grails-app/domain/org/pih/warehouse/order/OrderItem.groovy

```diff
@@ -119,7 +148,7 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
     }
 
     Integer quantityReceived() {
```

Comment:
What's the point of this method if quantityReceived is added to transients?

Reply:
So this is being used in a bunch of places and I don't feel like refactoring it all right now. The method that's in transients has no bearing on this method. I think this method was written before I understood transients and how they worked.